proto/metapb.proto

```diff
@@ -61,6 +61,42 @@ message RegionEpoch {
     uint64 version = 2;
 }
 
+message BucketStats {
```

Comment:
We will no longer count keys? Also, previously in region and store, queries were separable by type, but now only get and scan can be distinguished?